<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editar paciente</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 170, 255);
font: 75 12pt &quot;Sitka Heading&quot;;</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Heading</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Medidas Corporales</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Sitka Heading</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Peso: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="campo_peso_">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Sitka Heading</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Sitka Heading</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Estatura:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="campo_estatura_">
        <property name="font">
         <font>
          <family>Sitka Heading</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>371</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Heading</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Informaci√≥n </string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Sitka Heading</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Nombre: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="campo_nombre_">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Sitka Heading</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Sitka Heading</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Edad:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Sitka Heading</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Sexo:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Sitka Heading</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="campo_edad_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sitka Heading</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>17</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="campo_sexo_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sitka Heading</family>
          <pointsize>12</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Femenino</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Masculino</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>351</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="boton_editar">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(194, 236, 255);</string>
      </property>
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="examen">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(194, 236, 255);</string>
      </property>
      <property name="text">
       <string>Editar Examen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="salir">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(194, 236, 255);</string>
      </property>
      <property name="text">
       <string>Salir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
